### PR TITLE
Make textarea component

### PR DESCRIPTION
**Pull request by emmalearnscode**
Issue #100 Extract textarea from index file and make own component in styled folder

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import styled from 'styled-components';
-import TextareaAutosize from 'react-autosize-textarea';
 import useDimensions from 'react-use-dimensions';
 import { ConvertCard } from './styled/ConvertCard';
 import { IconX } from './IconX';
 import { IconCopy } from './IconCopy';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import { InMenuOptionStuff } from './InMenuOptionStuff';
 import { OutMenuOptionStuff } from './OutMenuOptionStuff';
-import { IInOption, InOptions, IOutOption, OutOptions } from './types';
```
Comment (igeligel) [inline src/index.tsx:11]: Hey, I think you should not have to delete the imports here. They are later exported:

```tsx
export { IInOption, IOutOption, InOptions, OutOptions };
```

And are necessary to have types for the users of this package.

**Review by igeligel — CHANGES_REQUESTED** [review]

**Comment by igeligel** [comment]
Looks mostly good for me. We need to change the imports/exports of types and then we can merge this Pull Request :)

**Comment by emmalearnscode** [comment]
Sure, my linter was complaining like crazy about the exports so I removed them while I was working on it and forgot to add them back in. Will fix it then let you know when it’s done

**Comment by emmalearnscode** [comment]
I added back in the imports/exports so this should be ready to go.

**Review by igeligel — APPROVED** [review]
Hey @emmalearnscode. Thanks for your contribution 👍 Would be nice if you could leave a star for the library.
I tagged this Pull Request with the necessary hacktoberfest labels, in case you are participating there. Thanks a lot, contributors like you make Open Source awesome 💯

I will also add you to the official contributor's list!

**Comment by igeligel** [comment]
@all-contributors please add @emmalearnscode for code

**Pull request merged**